### ISSUE TITLE
There is an error in the formula in the example of sorting by multiple columns.

### ISSUE THREAD

**Issue by koruneko** [MicrosoftDocs/power-platform#2042]
In the fifth example, you describe sorting by the `OnOrder` column and then by the `Quantity` column, but the `SortByColumns` formula is incorrect.  
The current description sorts by the `Quantity` column and then by the `OnOrder` column.  

incorrect(Current Description)  
```
Sort( Sort( IceCream, OnOrder ), Quantity )
```

```
SortByColumns( IceCream, "OnOrder", SortOrder.Ascending, "Quantity", SortOrder.Ascending )
```

correct  
```
Sort( Sort( IceCream, OnOrder ), Quantity )
```

```
SortByColumns( IceCream, "Quantity", SortOrder.Ascending, "OnOrder", SortOrder.Ascending )
```

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 57fbbefb-74dd-08c4-b6b2-6404ced0c689
* Version Independent ID: 57fbbefb-74dd-08c4-b6b2-6404ced0c689
* Content: [Sort and SortByColumns functions in Power Apps - Power Platform](https://learn.microsoft.com/en-us/power-platform/power-fx/reference/function-sort)
* Content Source: [power-platform/power-fx/reference/function-sort.md](https://github.com/MicrosoftDocs/power-platform/blob/main/power-platform/power-fx/reference/function-sort.md)
* Service: **power-platform**
* Sub-service: **power-fx**
* GitHub Login: @gregli-msft
* Microsoft Alias: **gregli**

**Comment by bishalgoswami** [comment]
Thanks for your comment, @koruneko. This issue has been assigned to the doc author for review.

**Comment by jorisdg** [comment]
@koruneko thanks for your feedback. I checked out this formula but it is correct.

`SortByColumns( IceCream, "OnOrder", SortOrder.Ascending, "Quantity", SortOrder.Ascending )`

sorts by OnOrder first, then by Quantity. If I try your suggestion, it will sort on Quantity first.

![image](https://github.com/MicrosoftDocs/power-platform/assets/16008040/9023d35b-ec49-418b-9dc3-98dd7923f71e)

**Comment by koruneko** [comment]
Here is the result of executing the equation currently listed as an example,

```
SortByColumns( IceCream, "OnOrder", SortOrder.Ascending, "Quantity", SortOrder.Ascending )
```

<img width="507" alt="image" src="https://github.com/MicrosoftDocs/power-platform/assets/40330021/29b3b06c-5ddb-41b7-8e58-87fecbceb08c">  

Here is the result of the run I proposed as a modified version.

```
SortByColumns( IceCream, "Quantity", SortOrder.Ascending, "OnOrder", SortOrder.Ascending )
```

<img width="510" alt="image" src="https://github.com/MicrosoftDocs/power-platform/assets/40330021/4a71500b-f282-46c7-9c2d-e135740d3ef0">  

The following images are used as output results in the example.  

![Result](https://learn.microsoft.com/en-us/power-platform/power-fx/reference/media/function-sort/icecream-onorder-quantity.png)  
https://learn.microsoft.com/en-us/power-platform/power-fx/reference/media/function-sort/icecream-onorder-quantity.png  

There seems to be a discrepancy between the formula described and its output results.  



> The current description sorts by the `Quantity` column and then by the `OnOrder` column.  

My statement of this was not correct.  
I feel sorry for the confusion.  

One suggestion would be to display the results of a sort on a table with duplicate values in the `Quantity` or `OnOrder` columns for better user understanding.  
I hope you will consider my proposal.

**Comment by jorisdg** [comment]
I agree the example is not very interesting because there are no duplicate values. It's a good idea and I will find some time to update the example with duplicates.
